talker2.cpp

```diff
@@ -0,0 +1,29 @@
+#include <ros/ros.h>
```

Comment:
Consider a way of doing this so that you dont have to make two files that are almost identical, look into some arguments you might be able to provide. http://wiki.ros.org/roslaunch/Commandline%20Tools